spec/libs/json_web_token_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'spec_helper'
```

Comment:
This is the problem spec - SignedJWT::create_portal_token expects a ENV variable for the hmac secret to exist.  I can set the ENV variable in the spec but it also needs to be set when the stubbed request is made so devise can check it and I can't figure out where to set that for rspec other than the app envionment variables file for test.